packages/api/src/store/rabbit-queue.ts

```diff
@@ -4,12 +4,21 @@ import { Channel, ConsumeMessage, Options } from "amqplib";
 
 const QUEUE_NAME = "webhook_default_queue";
 const EXCHANGE_NAME = "webhook_default_exchange";
+const WEBHOOK_CANNON_SINGLE_URL = "webhook_cannon_single_url";
+
+
 
 export default class MessageQueue {
   private channel: ChannelWrapper;
   private connection: AmqpConnectionManager;
+  private queues: Record<string, string>;
 
-  constructor() {}
+  constructor() {
+    this.queues = {
+      events: "webhook_default_queue",
+      webhooks: "webhook_cannon_single_url",
+    }
```

Comment:
**nit:** Since this is just a `const` in the end, WDYT of declaring it as a static const above and avoid repeating the queue names here? Say up there:
```
const Queues = {
      events: "webhook_default_queue",
      webhooks: "webhook_cannon_single_url",
    } as const
```

Then you could even type the `consume` function below more specifically, so it avoids typos in the consumer code as well:
```
  public async consume(queueName: keyof Queues, func: (msg: ConsumeMessage) => void): Promise<void> {
```